account/views.py

```diff
@@ -95,3 +95,12 @@ def post(self, request):
             user.save()
             return Response({"message": "Password Updated Successfully"}, status.HTTP_200_OK)
         return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+class ProfileView(APIView):
+    permission_classes = [IsAuthenticated]
```

Comment:
Remove this. There's already a default permission class; IsAuthenticatedOrReadOnly